modules/database.py

```
@@ -0,0 +1,104 @@
+import mysql.connector
+from mysql.connector import errorcode
+import logging
+
+
+class Mysql:
+    def __init__(self, db_host, db_user, db_password, db_name):
+
+        try:
+            cnx = mysql.connector.connect(
+                host=db_host,
+                user=db_user,
+                password=db_password,
+                database=db_name
+            )
+        except mysql.connector.Error as err:
+            if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+                logging.error("Something is wrong with your user name or password")
+            elif err.errno == errorcode.ER_BAD_DB_ERROR:
+                logging.error("Database does not exist")
+            else:
+                logging.error("Something went wrong at the connection :")
+                logging.error(err)
+            exit(1)
+
+        self.cursor = cnx.cursor()
+
+    def execute(self, sql, values):
+        try:
+            if not values or len(values) == 1:
+                self.cursor.execute(sql, values)
+            else:
+                self.cursor.executemany(sql, values)
+
+        except mysql.connector.Error as err:
+            if err.errno == errorcode.ER_TABLE_EXISTS_ERROR:
+                logging.error("Table already exists.")
+            elif err.errno == errorcode.ER_NO_SUCH_TABLE_ERROR:
+                logging.error("Table doesn't exist")
+            elif err.errno == errorcode.ER_WRONG_COLUMN_NAME_ERROR:
+                logging.error("Column doesn't exist")
+            else:
+                logging.error(err.msg)
+            logging.error(sql.format(values))
+            return 1
```

I'm not sure if you need to return something. maybe just re-raise the error